doc/source/tutorial/tutorial_presenting_larray_objects.ipyml

```diff
@@ -91,14 +91,40 @@ cells:
     axes = [age, sex, time, other]
     # data (the shape of data array must match axes lengths)
     data = np.random.randint(100, size=[len(axis) for axis in axes])
-    # title (optional)
-    title = 'random data'
+    # metadata
+    meta = [('title', 'random array')]
     
-    arr = LArray(data, axes, title)
+    arr = LArray(data, axes, meta=meta)
     arr
 
+  id: 4
+
+- markdown: |
+    Metadata can be added to an array at any time using: 
+
+
+- code: |
+    arr.description = 'array containing random values between 0 and 100'
+    
+    arr.meta
+
   id: 5
 
+- markdown: |
+    <div class="alert alert-warning">
+    
+    **Warning:**  
+      <ul>
+        <li>Currently, only the HDF (.h5) file format supports saving and loading metadata.</li>
+        <li>Metadata are not kept when actions or methods are applied on a array
```

Comment:
Data (and thus metadata I suppose) is most often considered singular (and uncountable) in English so most people would write "Metadata is" instead of "Metadata are"